virttest/qemu_vm.py

```diff
@@ -1192,12 +1192,15 @@ def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
                 logging.warn("-boot on s390x only support boot strict=on")
                 return "-boot strict=on"
             cmd = " -boot"
-            patterns = ["order", "once", "menu", "strict"]
+            patterns = ["order", "once", "menu", "strict", "reboot-timeout"]
+            boot_reboot_timeout = params.get('boot_reboot-timeout', 60)
```

Comment:
Suggest using the name `boot_reboot_timeout` for this parameter.

Reply:
Removed line1195~1196 and replaced  boot_reboot-timeout with boot_reboot_timeout in cfg seabios.cfg